src/role_harvester.js

```diff
@@ -20,6 +20,16 @@ roles.harvester.stayInRoom = true;
 roles.harvester.buildRoad = true;
 roles.harvester.boostActions = ['capacity'];
 
+roles.harvester.getPartConfig = function(room) {
```

Comment:
REMINDER: update prototype_room_utils.js need shorhand for
(room) => (room.storage && room.storage.my && room.storage.store.energy > config.creep.energyFromStorageThreshold)